source/unified-test-format/unified-test-format.rst

```diff
@@ -715,6 +715,10 @@ The structure of this object is as follows:
       This is to permit testing conditions where invalid KMS credentials are
       provided.
 
+      See the `Client-Side Encryption test README
+      <../client-side-encryption/tests/README.rst/credentials>`__ for
```

Comment:
This URL doesn't look correct. Should it be `#credentials`? Likewise below.

Reply:
> GitHub no longer appears to render the RST for this file

I noticed that. Seems to be the same issue some folks were reporting with some SDAM RST files. I had to pop the RST files in this PR into an RST preview tool to preview them.